misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java

```diff
@@ -370,10 +375,35 @@ private static String getMappingFromServer(JsonArray dataToSend,
                 }
             }
             JsonParser parser = new JsonParser();
-            return parser.parse(response.getData()).getAsJsonObject().get("answer").getAsString();
+            return parser.parse(response.getData()).getAsJsonObject().get("answer").toString();
         } catch (IOException e) {
             throw new IOException("Error connecting the AI service" + e.getMessage(), e);
         }
     }
-}
 
+    private static String generateMappingFunction(String mappingFromServer, String foundTypeLeft,
+                                                  String foundTypeRight, String leftModule, String rightModule) {
+
+        String leftType = foundTypeLeft;
+        String rightType = foundTypeRight;
+
+        mappingFromServer = mappingFromServer.replaceAll("\"", "");
```

Comment:
why do you need this?

Reply:
In the json response from the service, I get `""` for all the keys. So before generating the mapping function, Need to remove the `""` marks.